Dockerfile

```diff
@@ -14,6 +14,11 @@ RUN bin/bash -l -c "gem install aws-sdk mime-types"
 RUN apt-get install -y python2.7
 ENV PYTHON /usr/bin/python2.7
 
+# install hugo
+ENV HUGO_VERSION 0.21
```

Comment:
Just spoke with @jmhooper and we think best approach will be to include/use a default version, but provide a way for federalist users to specify a different version in their site settings. That UI needs to be determined. Some options:
-  text input box in Federalist web ui
- a `.hugo` file with a version string in the client repo
- something more "go"ish, like a `glide.yaml`